### ISSUE TITLE
File attachments: Name and tags fields not associated to label (1.3.1) 

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#844]
### Expected behavior

Label and field elements are programmatically related using for/id attributes or wrapping input in label element. 
### Actual behavior

There are label elements for the fields but they aren't connected to the input fields.
http://skyux.developer.blackbaud.com/components/fileattachments/

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.7.6](https://github.com/blackbaud/skyux/releases/tag/v1.7.6)
